README.md

```diff
@@ -37,6 +35,21 @@ Express.JS frameworks.
   operational delay can be enforced by using `DELAY` (in msec) environmental
   variable.
 
+* [x] `/events`
+  
+  ```bash
+  http :8080/events
+  ```
+  Returns a stream of Server-Sent Events, where each event is a CloudEvent 
+  represented in structured JSON format. This stream will continue to send next
+  events if they came.
+
+  ```bash
+  kn event send --field a.b=true --to-url http://localhost:8080/events
+  ```
+  You can send events to the app, by `POST /events` endpoint. Those events will
+  be stored in ephemeral im-memory storage and send to the listening clients.
```

Comment:
Maybe it makes sense to add `POST /` endpoint that will forward the handing to the `POST /events`. That would make it easier for customers, IMHO.

Reply:
Done in 5a4a52a16283e4c040c02d685507cb1afae3e54c